include/core/binary_reader.hpp

```diff
@@ -23,12 +23,12 @@ namespace rive
 
         size_t lengthInBytes() const;
 
-        uint64_t readVarUint();
         std::string readString();
         double readFloat64();
         float readFloat32();
         uint8_t readByte();
         uint32_t readUint32();
+        uint64_t readVarUint64(); // Reads a LEB128 encoded uint64_t
```

Comment:
Related to the issue, I just couldn't read the code and understand what type I'm supposed to write the format string for.

Reply:
I think this is fine for now as it gives a clear sense of what type to expect back, but yeah there might need to be a little more consideration on this one.